torchio/transforms/augmentation/intensity/random_ghosting.py

```diff
@@ -37,7 +37,6 @@ class RandomGhosting(RandomTransform, IntensityTransform):
             :math:`k`-space center should be restored after removing the planes
             that generate the artifact.
         p: Probability that this transform will be applied.
-        seed: See :py:class:`~torchio.transforms.augmentation.RandomTransform`.
```

Comment:
I remember getting lost with this seed parameter, deterministic transforms is definitively a better answer (than this previous seed argument)!

Reply:
I think so too! At least it's more explicit (which is better than implicit).

```python
import this
```